CHANGELOG.rst

```diff
@@ -495,7 +495,7 @@ October 26, 2016
 
 Bug Fixes
 ---------
-* Cython upgrade has broken stable version of cassandra-driver (PYTHON-656)
+* Cython upgrade has broken stable version of scylla-driver (PYTHON-656)
```

Comment:
Since you didn't publish that version under the new name, either keep it as is, or better yet, insert the fork point in the changelog where the history of `cassandra-driver` and `scylla-driver` diverges.

Reply:
yeah you got a point there... I've started with just blindly sed replacement...
this one should be reverted